chemie/events/views.py

```diff
@@ -36,12 +36,14 @@ def create_event(request):
 
 def list_all(request):
     all_events = Event.objects.filter(date__gt=timezone.now()).order_by('date')
+    past_events = Event.objects.filter(date__lte=timezone.now()).order_by('date')
     my_events = None
     if request.user:
         my_events = Event.objects.filter(attendees__username__exact=request.user)
     context = {
         'events': all_events,
         'my_events': my_events,
+        'past_events': past_events,
```

Comment:
Så en list_future, list_my_events, og en list_past_events? Ser for mæ at det blir nesten helt like views altså, og vi bruke fremdeles overview.html, no? Kan gjøre det.